samples/v1.2/Tests/ImageSizing.json

```diff
@@ -1,153 +1,187 @@
-                        {
-                            "type": "Image",
-                            "url": "https://picsum.photos/id/133/300/200",
-                            "size": "Large"
-                        },
-                        {
-                            "type": "Image",
-                            "url": "https://picsum.photos/id/133/300/200"
-                        },
-                        {
-                            "type": "Image",
-                            "url": "https://picsum.photos/id/133/300/200",
-                            "size": "Stretch"
-                        }
-                    ]
-                },
-                {
-                    "type": "Column",
-                    "width": "stretch",
-                    "items": [
-                        {
-                            "type": "TextBlock",
-                            "text": "Column 2"
-                        },
-                        {
-                            "type": "Image",
-                            "url": "https://picsum.photos/id/133/300/200",
-                            "size": "Small"
-                        },
-                        {
-                            "type": "Image",
-                            "url": "https://picsum.photos/id/133/300/200",
-                            "size": "Large"
-                        },
-                        {
-                            "type": "Image",
-                            "url": "https://picsum.photos/id/133/300/200"
-                        },
-                        {
-                            "type": "Image",
-                            "url": "https://picsum.photos/id/133/300/200",
-                            "size": "Stretch"
-                        }
-                    ]
-                }
-            ]
-        },
-        {
-            "type": "ImageSet",
-            "images": [
-                {
-                    "type": "Image",
-                    "url": "https://picsum.photos/id/133/300/200"
-                }
-            ]
-        },
-        {
-            "type": "ImageSet",
-            "imageSize": "Small",
-            "images": [
-                {
-                    "type": "Image",
-                    "url": "https://picsum.photos/id/133/300/200"
-                }
-            ]
-        },
-        {
-            "type": "ImageSet",
-            "imageSize": "Medium",
-            "images": [
-                {
-                    "type": "Image",
-                    "url": "https://picsum.photos/id/133/300/200"
-                }
-            ]
-        },
-        {
-            "type": "ImageSet",
-            "imageSize": "Large",
-            "images": [
-                {
-                    "type": "Image",
-                    "url": "https://picsum.photos/id/133/300/200"
-                }
-            ]
-        }
-    ]
-}
\ No newline at end of file
+  "type": "AdaptiveCard",
```

Comment:
I can't access codeflow, but if these lines are changing because it converting from single spaces to tabs then ignore this comment, otherwise revert to tabs

Reply:
Done. Converted all whitespace to tabs.